SafeSurroundings/SafeSurroundings/Views/Shared/_ProfileAvatar.cshtml

```diff
@@ -1,8 +1,8 @@
 ﻿<link href="~/Content/GeneralStyle.css" rel="stylesheet" />
 <div>
-    @if ((Session.Keys.Count > 0) && (!String.IsNullOrEmpty(Convert.ToString(Session["sessionGuid"]))))
+    @if ((Session.Keys.Count > 0) && (!String.IsNullOrEmpty(Convert.ToString(Session["sessionGuid"]))) && (!String.IsNullOrEmpty(Convert.ToString(Session["avatarSrc"]))))
     {
-        <img src="~/Images/dog.png" class="navBar-brand-profile-picture" />
+        <img src="@Url.Content(Convert.ToString(Session["avatarSrc"]))" alt="avatar" class="navBar-brand-profile-picture" />
     }
```

Comment:
Maybe add a catch-all or a default static image when there is not image to display.

Reply:
Added a default catch-all static image for when the src is invalid, no image for a logged-in user. (Deleted image post login,etc). 

This might be an idea for a set of extension functions to use in the view.
![image](https://user-images.githubusercontent.com/26256313/145481777-c7d602fc-289d-4be4-9e7b-e7bd350477d7.png)